zkg

```diff
@@ -453,9 +469,13 @@ def cmd_test(manager, args, config, configfile):
     package_infos = []
 
     for name in args.package:
-        if is_git_repo_dirty(name):
-            print_error('error: local git clone at {} is dirty'.format(name))
-            sys.exit(1)
+        if is_local_git_repo_url(name):
+            if not is_local_git_repo(name):
+                print_error('error: path {} is not a git repository'.format(name))
+                sys.exit(1)
+            if is_local_git_repo_dirty(name):
+                print_error('error: local git clone at {} is dirty'.format(name))
+                sys.exit(1)
```

Comment:
How about one more wrapper function for this block, which is repeated 3 times across the commands.

Reply:
Yeah good point, thanks Robin. Done. We may want to start a `zeekpkg/_gitutil.py` soon, just for all the git-related utility functions...